.github/workflows/ci.yaml

```diff
@@ -30,4 +30,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v3
-    - run: docker build --file e2e2/test/images/nvidia/Dockerfile .
\ No newline at end of file
+    - run: docker build --file e2e2/test/images/nvidia/Dockerfile .
+  build-bert-inference:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - run: docker build --file e2e2/test/images/bert-inference/Dockerfile e2e2/test/images/bert-inference
```

Comment:
as much as i prefer setting the build workspace to the exact path, I think it's set to `.` in the other images because its built from the root in other workflows. Can you double check this will work

Reply:
This doesn't work because the image build references `e2e2/test/images/bert-inference/requirements.txt`. If we set the path to `.`, instead of the full path, then it doesn't know it exists. This is actually something I was wanting to discuss, python dependency declaration/management, but was hoping it'd be a down the line thing (not a short convo)

Reply:
i agree, i don't love the full path but we'll need to make sure it works with our current setup